code/modules/projectiles/guns/misc/syringe_gun.dm

```diff
@@ -61,6 +61,9 @@
 
 /obj/item/gun/syringe/attackby(obj/item/A, mob/user, params, show_msg = TRUE)
 	if(istype(A, /obj/item/reagent_containers/syringe))
+		if(istype(A, /obj/item/reagent_containers/syringe/lethal) || istype(A, /obj/item/reagent_containers/syringe/bluespace)) //monkestation edit: no more lethal or bluespace syringes
```

Comment:
making it a var on the syringe is probably better

Reply:
Good suggestions!  Both implemented.